app/models/hostgroup.rb

```diff
@@ -113,8 +113,15 @@ def self.title_name
     "title".freeze
   end
 
+  def disk_layout_template
+    if ptable.present?
+      { name: ptable.name, template: ptable.layout }
+    end
+  end
+
   def diskLayout
-    ptable.layout.tr("\r", '')
+    raise 'Partition table not defined for hostgroup' unless disk_layout_template
```

Comment:
Should this raise a `Foreman::Renderer::RenderingError`?

Reply:
Updated